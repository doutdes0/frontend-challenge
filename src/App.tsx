import { Outlet } from "react-router-dom";
import NavBar from "./components/Navbar";
import styled from "styled-components";

const App: React.FC = () => {
  return (
    <Container>
      <NavBar />
      <Outlet />
    </Container>
  );
};

export default App;

const Container = styled.div`
  min-height: 100vh;
  position: relative;
  display: grid;
  grid-template-rows: 64px auto;
`;
