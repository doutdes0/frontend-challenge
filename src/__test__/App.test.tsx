import { BrowserRouter } from "react-router-dom";
import App from "../App";
import { render } from "@testing-library/react";

describe("App", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <BrowserRouter>
        <App />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
