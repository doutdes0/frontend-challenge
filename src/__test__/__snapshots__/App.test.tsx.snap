// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div>
  <div
    class="sc-guDLey hRbNZB"
  >
    <div
      class="sc-beySPh kqHCnd"
    >
      <div
        class="link active"
        data-link="/"
      >
        <a
          aria-current="page"
          class="active"
          href="/"
        >
          Все котики
        </a>
      </div>
      <div
        class="link"
        data-link="/favorites"
      >
        <a
          class=""
          href="/favorites"
        >
          Любимые котики
        </a>
      </div>
    </div>
  </div>
</div>
`;
